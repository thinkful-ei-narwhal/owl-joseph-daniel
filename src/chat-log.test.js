import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ChatLog from './chat-log';


describe('<ChatLog />', () => {
  it('smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ChatLog chatEvents ={[{
      participantId: 2,
      type: 'thumbs-up',
      timestamp: 1548852484247
  },
  {
      participantId: 4,
      type: 'thumbs-down',
      timestamp: 1548852484247
  },]}/>, div);
    // if something is wrong, itd fail here
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create(<ChatLog chatEvents ={[{
      participantId: 2,
      type: 'thumbs-up',
      timestamp: 1548852484247
  },
  {
      participantId: 4,
      type: 'thumbs-down',
      timestamp: 1548852484247
  },]}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});