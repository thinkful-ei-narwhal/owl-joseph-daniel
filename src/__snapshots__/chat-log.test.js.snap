// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatLog /> renders this UI as expected 1`] = `
<div
  className="chat-list"
>
  <hr />
  <div
    className="Chat-event"
  >
    <img
      alt="Caty Flucker"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <h3>
      Caty Flucker
    </h3>
    <p>
      
    </p>
    <p>
      
    </p>
    <p>
      gave a thumbs up
    </p>
  </div>
  <div
    className="Chat-event"
  >
    <img
      alt="Frank Runciman"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <h3>
      Frank Runciman
    </h3>
    <p>
      
    </p>
    <p>
      
    </p>
    <p>
      gave a thumbs down
    </p>
  </div>
</div>
`;
